quic_io/readme.md

```diff
@@ -0,0 +1,24 @@
+## QUIC RAW Library
+
+This library is based on chromium source code and exposes basic interfaces for raw data transporting.
+
+## How to build
+1. Get the chromium code. (https://chromium.googlesource.com/chromium/src/+/master/docs/linux_build_instructions.md)
```

Comment:
This library is not only for Linux. Please restructure for Win/Linux  for instructions different on platform and about build/use.